packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -66,8 +66,8 @@ export interface ServerlessClusterProps {
   readonly clusterIdentifier?: string;
 
   /**
-   * The number of days during which automatic DB snapshots are retained. Set
-   * to zero to disable backups.
+   * The number of days during which automatic DB snapshots are retained.
+   * Automatic backup retention cannot be disabled on serverless clusters.
```

Comment:
So out of curiosity, what happens if I pass `Duration.days(0)` here?

Reply:
ah whoops, i forgot to commit the changes I made to add validation. It would fail deployment and whine about 0 being an invalid value. let me add my validation and tests back in by unwinding my stashes 🙂